templates/jira/Epic.md

```diff
@@ -0,0 +1,31 @@
+# Epic ticket template
+
+An Epic is a big feature or task. It can span multiple sprints and can be broken down into smaller tasks. Use Epic tickets to group related tasks together into logical collections.
```

Comment:
Can we quantify "big"? Or give some examples?

Reply:
Added "(spans multiple sprints)" see: https://github.com/pondevelopment/pon-developer-guide/commit/ef14329caf5859cef42c096e41348600cc568590